etc/kube/pachyderm.json

```diff
@@ -343,7 +343,10 @@
       "spec": {
         "volumes": [
           {
-            "name": "pach-disk"
+            "name": "pach-disk",
+            "hostPath": {
+              "path": "/tmp/pach-402374041"
```

Comment:
It's generated from pach-deploy.  It's just a random directory.  Running pach-deploy again will result in a different directory.